eval/context.go

```diff
@@ -161,8 +161,12 @@ func (c *Context) WithClientRequest(req *http.Request) *Context {
 		}
 	}
 	port, _ := strconv.ParseInt(p, 10, 64)
-	// TODO how to set request.BufferOptions appropriately before WithClientRequest() is called
```

Comment:
@johakoch this commit should answer this question. Please do a quick review :)

Reply:
Looks ok. Do we need a test like TestHTTPServer_Endpoint_Response_JSONBody_Evaluation _without_ reference to request.json_body showing that the request body is _not_ json-parsed?

Reply:
I added another test case.